static/js/components/property/action.js

```diff
@@ -77,20 +79,18 @@ template.innerHTML = `
       display: inline-block;
     }
   </style>
-  <div id="container" class="webthing-action-container">
-    <div id="contents" class="webthing-action-contents">
-      <button id="button" type="button" class="webthing-action-button"></button>
+  <div id="container-${BaseComponent.count}" class="webthing-action-container">
```

Comment:
Since all of these elements are retrieved using `this.shadowRoot.querySelector` I believe a better way of solving the ID conflicts is to use querySelector with the class instead of the id and drop the id altogether.